### ISSUE TITLE
Windows command & customization

### ISSUE THREAD

**Issue by isteven** [7ute/symbols-list#8]
Hi, 

Thanks for the great plugin, it really helps me!

A small bug; On the plugin settings page, I saw the default Mac toggle, which is **alt-cmd-l**. On Windows this would translate into alt-ctrl-l, but somehow it doesn't work. I know this can be configured manually, but I thought I'd just let you know.

Would love to see further customization if possible, example:
- Only toggle on certain file types
- What symbols to show: object, variable, etc.

Thanks & keep up the good work!

**Comment by 7ute** [comment]
Hi,
Thank you for your feedback :)

1/ Yeah, i should probably add some linux/windows dedicated shortcut. If alt-ctrl-l ain't taken already, that could do it. I don't have Atom installed on a windows machine so if you can confirm that this shortcut is free on vanilla atom, I'll add it.
2/ Ah yes, I noticed that one. It may have to do with some regexes matching line starting carriage returns ^^'. I'll fix that in next release.

As for the fine tuning, I was thinking of something like that.
Maybe some sort of hierarchical table having sources types as parents, and symbols as children, and checkboxes to activate/deactivate sources or symbols.
But that's a bit heavy for now, and by keeping the symbol list light, we may avoid the need of such settings? :)

Cheers!

**Comment by isteven** [comment]
Hi, 

So far, I can confirm that no default Atom command using alt-ctrl-l / ctrl-alt-l. So it's still free.

Instead of a hierarchical table, perhaps you can just provide a simple textfield so people can just put a space-separated file types, such as:

- Only toggle on these file extensions: ".php .js .css .java". 

Same for the symbols to show/hide, such as:

- Hide these symbols: "inline-function object array variable". 

So basically they can be whitelist or blacklist, it's up to you. (However, for the symbols show/hide, you need to tell people what's settings are available).

Also, perhaps a simple legend is needed for the square icons you put in front of each function names. I'm a Javascript developer, and I saw "F" and "M". I'm sure "F" means function, but what is "M"?

Cheers.

**Comment by 7ute** [comment]
"M" is for "Method" :)
1/ and 2/ Fixed on 2.2 release

Ill keep that ticket open, to get ideas when refactoring the settings!

**Comment by isteven** [comment]
Oh M for method.. silly me didn't realize it in the first place.

Thank you for the fixes, much appreciated, and, I look forward to the settings customization :)

Cheers.

**Comment by isteven** [comment]
Hi @7ute, 

I have tested & I can confirm that the Windows shortcut is now working OK.

However, unfortunately problem no. 2) still exists. Also, upon further checking, it seems that it only happens on Functions (F). The Methods (M) seem to be spot on, at least from what I tested.

Cheers.

**Comment by 7ute** [comment]
Oh that's true, thanks for noticing it!
I forgot to fix that "jump to line" bug on JS. It seems to be ok for PHP though.
It'll be done for next release :)
